AppManager/app_manager_server.py

```diff
@@ -90,9 +93,16 @@
 PIDFILE_TEMPLATE = os.path.join('/', 'var', 'run', 'appscale',
                                 'app___{project}-{port}.pid')
 
+# The number of seconds an instance is allowed to finish serving requests after
+# it receives a shutdown signal.
+MAX_INSTANCE_RESPONSE_TIME = 600
```

Comment:
Are you referring to the constant I also added in the SDK? I want to avoid importing anything from the SDK (or importing from an appscale package to the SDK) since it makes it more difficult to separate from the rest of our codebase.